presto-docs/src/main/sphinx/functions/string.rst

```diff
@@ -56,6 +56,16 @@ String Functions
 
     Removes trailing spaces from ``string``.
 
+.. function:: split(string, delimiter) -> array<varchar>
+
+    Splits ``string`` on ``delimiter`` and return an array.
+
+.. function:: split(string, delimiter, limit) -> array<varchar>
+
+    Splits ``string`` on ``delimiter`` and return an array of size at most
+    ``limit``. The last element in the array always contain everything
```

Comment:
Hmm, this seems weird to put everything in the last element. Check with @electrum or @martint if that's how it should work